WooCommerce/Classes/Extensions/NSMutableAttributedString+Helpers.swift

```diff
@@ -13,4 +13,24 @@ extension NSMutableAttributedString {
             addAttributes(attributes, range: range)
         }
     }
+
+    /// Replaces the first found occurrence of `target` with the `replacement`.
+    ///
+    /// Example usage:
+    ///
+    /// ```
+    /// let attributedString = NSMutableAttributedString(string: Hello, #{person}")
```

Comment:
super nit: the string value is missing a front quote `string: "Hello, #{person}"`